pkg/scheduler/framework/plugins/default_registry.go

```diff
@@ -29,9 +30,7 @@ import (
 // runs custom plugins, can pass a different Registry when initializing the scheduler.
 func NewDefaultRegistry() framework.Registry {
 	return framework.Registry{
-		// This is just a test plugin to showcase the setup, it should be deleted once
-		// we have at least one legitimate plugin here.
-		noop.Name: noop.New,
+		tainttoleration.Name: tainttoleration.New,
```

Comment:
A question here: do we have a mechanism to keep a fixed order of the same type of plugins (e.g., FilterPlugin)

Reply:
yes, this order is determined in the [config.Plugins](https://github.com/kubernetes/kubernetes/blob/master//pkg/scheduler/apis/config/types.go#L148:6) we pass to the framework when it is created.

For Filters that get mapped from predicates, they are added according to the predicate ordering: https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/factory/factory.go#L557